packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -385,7 +385,45 @@ class ShortcutManager extends ChangeNotifier with Diagnosticable {
 /// A widget that establishes an [ShortcutManager] to be used by its descendants
 /// when invoking an [Action] via a keyboard key combination that maps to an
 /// [Intent].
-///
+/// 
```

Comment:
Remove extra spaces at the end of lines.

Reply:
Thanks, @yash1200 😄